apidoc/Titanium/Android/Activity.yml

```diff
@@ -294,6 +294,14 @@ events:
         type: Titanium.Android.Intent
     since: '3.0.0'
 
+  - name: onIntent
```

Comment:
@sgtcoolguy @cheekiatng thoughts please. I will merge it for now so we have it in the docs, but we should file a ticket to migrate all "on<event-name>" events to proper names so they don't conflict with Alloy and our usual naming conventions regarding events.